src/features/users/usersSlice.ts

```diff
@@ -0,0 +1,51 @@
+import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit';
+import { User } from '../../types/User';
+import { getUsers } from '../../api/users';
```

Comment:
Verify that the `getUsers` function is correctly implemented and imported from the `api/users` file.